src/java/org/apache/cassandra/service/accord/txn/TxnWrite.java

```diff
@@ -261,7 +261,7 @@ public Update complete(AccordUpdateParameters parameters)
             if (!staticRow.isEmpty())
                 updateBuilder.add(staticRow);
 
-            Row existing = !baseUpdate.isEmpty() ? Iterables.getOnlyElement(baseUpdate) : null;
+            Row existing = baseUpdate.hasRows() ? Iterables.getOnlyElement(baseUpdate) : null;
```

Comment:
found by `TopologyMixupTest`. The test used `UPDATE` on a row that didn't exist, but the table had a static column... the query was

```
UPDATE ...
SET
  a += 42 -- row doesn't exist, null + 42 = null
  b = 7 -- this is a static column, so creates an update
```

the update had a static row updated, but no non-static row!

Reply:
maybe it would be good to be explicit about this in one of our tests rather than rely on a fuzz test... regression tests are great to avoid the same bugs happening again...

I should add...

Reply:
wrote the test `org.apache.cassandra.distributed.test.accord.AccordCQLTestBase#testNonExistingKeyWithStaticUpdate`